utils/account-setup/index.js

```diff
@@ -120,7 +120,7 @@ async function main() {
         const purchaser_lines = await newAccounts[4].xrplAcc.getTrustLines(EvernodeConstants.EVR, newAccounts[0].xrplAcc.address);
 
         if (purchaser_lines.length === 0) {
```

Comment:
Shall we remove this trunstline addition for the Purchaser Hot wallet?
Also, it is better if you can remove the accounts that are not used in the `accounts` array and modify the indices where necessary (`newAccounts`).